### DISCUSSION TITLE
zeek logs after fresh install

### DISCUSSION THREAD

**Discussion by Bikesh-Yadav** [Security-Onion-Solutions/securityonion#10183] [General]
Hello,
i have installed SO 2.3.23 in my VM, while installing i have selected Zeek for logs but i didn't see any zeek logs.

outputs:
1. Docker image output
root@ubuntu:~# sudo docker images | grep zeek
ghcr.io/security-onion-solutions/so-zeek                     2.3.230   78d8618de9dc   12 days ago     2.18GB
ubuntu:5000/security-onion-solutions/so-zeek                 2.3.230   78d8618de9dc   12 days ago     2.18GB

2.mdengine output

salt-call pillar.get global:mdengine
local:
    ZEEK

3. so-zeek-start  
      [ERROR   ] Failure!
local:
----------
          ID: zeek_state_not_allowed
    Function: test.fail_without_changes
      Result: False
     Comment: Failure!
     Started: 10:48:38.711003
    Duration: 1.109 ms
     Changes:
..
is there any changes or modifications needed to get zeek logs?

**Comment by cm-ops** [comment]
What type of node are you running `so-zeek-start` on? `salt-call grains.get id`

**Reply by Bikesh-Yadav** [reply]
I am running this command: "so-zeek-start" on Managersearch node

**Comment by cm-ops** [comment]
Zeek is not permitted to run on a Managersearch node, you would need a node with sensor components running.
